### PR TITLE
Prepare 1.27.0 release.

### PR DESCRIPTION
**Pull request by nc6**

### CODE REVIEW

**Review by LaurenceIO — APPROVED** [review]
Looks good to me, thanks Nick!

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

File: cardano-cli/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+## 1.27 -- April 2021
+
+- The query tip now also returns the era (e.g. Shelley, Allegra, Alonzo).
+  (#2561, #2562, #2598)
+- The `address build` command now incorporates the functionality of the script
+  address build command, which is now deprecated. (#2486, #2587)
+- Add additional commands for creating MIR certificates to the CLI. This
+  supports the ability to transfer funds to the treasury for Catalyst projects.
+  (#2503)
+- The transaction view command now additionally shows detailed of minted
```
Comment (huths0lo) [inline cardano-cli/ChangeLog.md:12]: typo on this line.  Should be "details"

**Pull request merged**